controller/sync_test.go

```diff
@@ -5,13 +5,16 @@ import (
 	"sort"
 	"testing"
 
+	"github.com/argoproj/argo-cd/reposerver/repository"
```

Comment:
Can you move this import into the internal project import section at line 22?

Reply:
Sure, done: https://github.com/argoproj/argo-cd/pull/979/commits/1a816900d6ab9469725cb40d9d5f969e08eef095